core/src/main/scala/kafka/admin/BrokerApiVersionsCommand.scala

```diff
@@ -39,7 +40,17 @@ object BrokerApiVersionsCommand {
   def execute(args: Array[String], out: PrintStream): Unit = {
     val opts = new BrokerVersionCommandOptions(args)
     val adminClient = createAdminClient(opts)
-    val brokerMap = adminClient.listAllBrokerVersionInfo()
+
+    // Wait until AdminClient#findAllBrokers returns a non-empty list of brokers
```

Comment:
Just my 2 cents. I actually think this logic is waiting is better here unless we have more use-case for this `awaitBrokers()`. The idea is to focus AdminClient on the admin operation.

I am wondering if the code below may be slightly better:
```
var brokerMap = Map.empty[Node, Try[NodeApiVersions]]

do {
  Thread.sleep(50)
  brokerMap = adminClient.listAllBrokerVersionInfo()
} while (brokerMap.isEmpty)
```